scripts/auto-rebase/rebase.sh

```diff
@@ -904,18 +1054,25 @@ usage() {
 command=${1:-help}
 case "$command" in
     to)
-        [[ $# -ne 3 ]] && usage
-        rebase_to "$2" "$3"
+        [[ $# -ne 4 ]] && usage
+        rebase_to "$2" "$3" "$4"
```

Comment:
If we're changing the usage, let's update the "usage" function by adding something like LVMS_IMAGE_MULTI_ARCH?
We would also need to update the rebase automation calling this script.